utils/cros-vboot/Makefile

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (C) 2020 Brian Norris <computersforpeace@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=cros-vboot
+PKG_VERSION:=20200715
```

Comment:
If `PKG_VERSION` is not specified, the build system will derive a version of the form `netifd-2020-06-06-51e9fb81.tar.xz`.  More informative and familiar if we just stick to that.

Reply:
https://github.com/openwrt/openwrt/blob/master/include/download.mk#L14